### ISSUE TITLE
[FEATURE] Music Control Panel (as text-channel in Discord app)

### ISSUE THREAD

**Issue by CtrlAltSpoods** [SudhanPlayz/Discord-MusicBot#1173]
**Is your feature request related to a problem? Please describe.**
A nice feature that Hydra has is that it has the option to create a text-channel entirely dedicated to music requests, now that Hydra no longer plays any music at all it is a highly missed feature in thousands of Discord servers.

**Describe the solution you'd like**
I think it'd be a great idea to implement this as it would attract potentially thousands of new bot installs.
- The UI can be similar to that of Hydra (pictured below)
- The UI has buttons for controlling the media.
- The bot also uses new messages in this chat as a search command/direct url depending on what the user sends, then adds this media into the queue.
- It also will pull sound files like MP3s/WAVs and play them when uploaded.
- All chats are deleted instantly after media has been queued.
- Describe alternatives you've considered
- An alternative is a web panel, but in my experience, this is a bit more finnicky to use while trying to game. (The text-channel approach is much simpler and easier for non-tech savvy users)

**Additional context**
I consider this low-priority and a nice-to-have, not 100% necessary, but I do believe it will certainly attract MANY new users to this bot.

![image](https://user-images.githubusercontent.com/123032706/219714257-ea559f51-fa42-48fd-881a-7b8d8b43e429.png)

**Comment by Assassin654** [comment]
What version of the bot are you using. It already has buttons, deletes messages etc. If you want a channel for the bot to only talk in; That is something you can do too just through discord.

**Comment by CtrlAltSpoods** [comment]
> What version of the bot are you using. It already has buttons, deletes messages etc. If you want a channel for the bot to only talk in; That is something you can do too just through discord.

Version 5, I see that the bot has controls for playing and pausing etc, and I have already set it up to only allow the slash commands in one channel.
What I mean is this dashboard looking UI w/ logo and queue that appears when songs are playing, it updates the embed dynamically and removes all users posts after they have been added to keep the channel clean and neat.

If I submitted a post saying `Daft Punk - Around The World` it would search for it, queue/play the song and delete my request post automatically.
